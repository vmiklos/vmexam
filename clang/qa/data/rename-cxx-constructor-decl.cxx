class C
{
public:
    C();
    int nX;
};

C::C()
     : nX(0)
{
}
