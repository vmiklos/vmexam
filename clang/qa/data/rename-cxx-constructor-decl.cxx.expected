class C
{
public:
    C();
    int m_nX;
};

C::C()
     : m_nX(0)
{
}
